prusti-specs/src/rewriter.rs

```diff
@@ -116,70 +96,105 @@ impl AstRewriter {
             &format!("prusti_{}_item_{}_{}", spec_type, item.sig().ident, spec_id),
             item_span,
         );
-        let mut statements = TokenStream::new();
-        assertion.encode_type_check(&mut statements);
         let spec_id_str = spec_id.to_string();
-        let assertion_json = crate::specifications::json::to_json_string(&assertion);
 
-        let mut spec_item: syn::ItemFn = parse_quote_spanned! {item_span=>
+        let mut spec_item: syn::ItemFn = parse_quote_spanned! {item_span =>
```

Comment:
For the record this weird spacing is suggested [here](https://docs.rs/quote/1.0.9/quote/macro.quote_spanned.html#syntax) but I don't find the justification very good. I haven't been consistent about the spacing and probably wrote the space before `=>` more often than not.

Reply:
Right. Fixed by 681ccfe0b2aa35aa9ed35a05b55490ba69ad34b9